Framework/Geometry/src/Instrument/FitParameter.cpp

```diff
@@ -4,8 +4,10 @@
 #include "MantidGeometry/Instrument/FitParameter.h"
 #include "MantidGeometry/Instrument/Parameter.h"
 #include "MantidGeometry/Instrument/ParameterFactory.h"
-#include <MantidKernel/StringTokenizer.h>
 #include "MantidGeometry/muParser_Silent.h"
+#include "MantidKernel/StringTokenizer.h"
+
+#include <boost/lexical_cast.hpp>
```

Comment:
Do we want quotes or angle brackets for third-party libraries? My preference has been for the former (leaving angle brackets for just the standard library), but the opposite isn't wrong and ultimately what matters is consistency.